packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js

```diff
@@ -41,11 +45,18 @@ const AddressBookPage = props => {
                     onClick={handleAddAddress}
                 >
                     <Icon
-                        classes={{ icon: classes.addIcon }}
+                        classes={{
+                            icon: classes.addIcon
+                        }}
                         size={24}
                         src={PlusSquare}
                     />
-                    <span className={classes.addText}>{'Add an Address'}</span>
+                    <span className={classes.addText}>
+                        <FormattedMessage
```

Comment:
If we're already using the hook do we need to also use FormattedMessage? Not holding the PR on this but just curious what you think.

Reply:
I think it's good to have options on different ways to implement `react-intl`